### PR TITLE
Correcting Format Issues present in DG

### PR DESCRIPTION
**Pull request by shxr3f**
Changing puml files to png files.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -90,33 +90,42 @@ How the Manager class works:
 * The while loop only halts when isTerminated boolean becomes true. Then, the programme exits
 
 ### Helper Classes
+
 #### `Command`
 
-![CommandUML](https://raw.githubusercontent.com/AY2122s2-cs2113t-t09-3/tp/master/docs/Diagrams/CommandClassUML.png)
+![CommandUML](diagrams/CommandClassUML.png)
+
```
Comment (tanweili) [inline docs/DeveloperGuide.md:97]: I think add <br> under all the .png to ensure that the para is below

**Review by tanweili — CHANGES_REQUESTED** [review]
A bit anal but some paragraphs do not have full stops. So just this and the <br> and I think good to go.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (shxr3f) [reply docs/DeveloperGuide.md:97]: Made adjustments in my latest commit.

**Review by tanweili — APPROVED** [review]
Good to go!

**Pull request merged**